.travis.yml

```diff
@@ -36,19 +40,31 @@ matrix:
       jdk: oraclejdk8
       android:
         components:
-          - android-28
-          - build-tools-28.0.3
+          - tools
+          - android-25
+          - android-26
+          - build-tools-26.0.3
       env:
         - PLATFORM_NAME=android
         - RELEASE_ZIP_FILE=${MODULE_ROOT}/${PLATFORM_NAME}/dist/${MODULE_NAME}-${PLATFORM_NAME}-${TRAVIS_TAG}.zip
+        - EMULATOR_IMAGE="system-images;android-25;google_apis;armeabi-v7a"
+        - ANDROID_HOME=/usr/local/android-sdk
+        - PATH=${ANDROID_HOME}/emulator:${PATH}
       before_install:
+        - nvm install --lts
+        - nvm use --lts
+        - npm install -g titanium
         - echo y | sdkmanager "ndk-bundle"
+      before_script:
+        - sdkmanager --list
+        - echo yes | sdkmanager "tools"
+        - echo yes | sdkmanager "${EMULATOR_IMAGE}"
+        - echo no | avdmanager create avd -n test -k "${EMULATOR_IMAGE}" -f --abi armeabi-v7a --tag google_apis
+        - emulator -avd test -engine classic -no-window -camera-back none -camera-front none -verbose -qemu -m 512 &
+        - android-wait-for-emulator
+        - adb shell input keyevent 82 &
```

Comment:
Why not just use the Jenkins module pipeline? :D

Reply:
#13 happy now? 🤣

Reply:
Getting better 😁